operators/src/upgrade/common/error.rs

```diff
@@ -0,0 +1,18 @@
+/// Helm error to be used for client.
```

Comment:
why is everything pub?
Is the the upgrade operator a library or a binary?

Reply:
Upgrade operator is a binary. Have updated the code.